plugins/person-extras/src/components/SkillsEditor.svelte

```diff
@@ -0,0 +1,73 @@
+<!--
+Copyright © 2020, 2021 Anticrm Platform Contributors.
+
+Licensed under the Eclipse Public License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may
+obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<script lang="ts">
+  import workbench from '@anticrm/workbench/src/__model__'
+
+  import Icon from '@anticrm/platform-ui/src/components/Icon.svelte'
+  import SkillEditor from './SkillEditor.svelte'
+  import personExtras, { Skill } from '..'
+
+  export let skills: Skill[]
+
+  function addSkill() {
+    skills = [
```

Comment:
Why not use push? Did Svelte had some issues with it?

Reply:
At least that didn't work to me, I'm not svelte expert but from `React` perspective that's quite reasonable to rely on reference to update tree.

Reply:
Just curious, since I've did some push operations and it worked, just will need to check if any existing code uses it wrong.

Reply:
Yeah ok, I'll take a closer look

Reply:
Per doc it is required to assign target variable to make svelte to update: https://svelte.dev/tutorial/updating-arrays-and-objects

Reply:
ok, will ensure I've use this approach, Thanks.